health/warnings.go

```diff
@@ -103,11 +103,15 @@ var noDERPHomeWarnable = Register(&Warnable{
 // noDERPConnectionWarnable is a Warnable that warns the user that Tailscale couldn't connect to a specific DERP server.
 var noDERPConnectionWarnable = Register(&Warnable{
 	Code:      "no-derp-connection",
-	Title:     "No relay server connection",
+	Title:     "Relay server unavailable",
 	Severity:  SeverityHigh,
 	DependsOn: []*Warnable{NetworkStatusWarnable},
 	Text: func(args Args) string {
-		return fmt.Sprintf("Tailscale could not connect to the relay server '%s'. The server might be temporarily unavailable, or your Internet connection might be down.", args[ArgRegionID])
+		if regionName := args[ArgRegionName]; regionName != "" {
```

Comment:
this line says "regionName" three times, which is a bit over 50% of the characters on the line.

This is where the "short scopes can have short variable names" thing comes into play. The scope of this "regionName" variable is two lines, if you could this line. It's okay to call this "n". The reader will know which "n" you're talking about.

Reply:
fixed